torchmetrics/__init__.py

```diff
@@ -63,7 +63,7 @@
     RetrievalPrecision,
     RetrievalRecall,
 )
-from torchmetrics.text import WER, BERTScore, BLEUScore, ROUGEScore  # noqa: E402
+from torchmetrics.text import WER, BERTScore, BLEUScore, ROUGEScore, SacreBLEUScore  # noqa: E402
```

Comment:
```suggestion
from torchmetrics.text import BERTScore, BLEUScore, ROUGEScore, SacreBLEUScore, WER  # noqa: E402
```

Reply:
This order of imported modules is actually enforced by the pre-commit hook